Initial Contribution (#20)
MrLongbottom/P10-Article: sections/introduction.tex

	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
	\item \textit{How can multiple metadata fields be integrated within a topic model?}
\end{itemize}

> Our work is similar to that of \citet{MetaLDA2017}, since we working with the meta information and combining it into an \gls{lda} model.

Reviewer: ```suggestion
Our work is similar to that of \citet{MetaLDA2017}, since we work with meta information and apply it into an \gls{lda} model.
```

Author: ```suggestion
Our work is similar to that of \citet{MetaLDA2017} since we work with meta information and applying it into an \gls{lda} model.
```

Reviewer: ```suggestion
Our work is similar to that of \citet{MetaLDA2017} since we work with meta information and are applying it into an \gls{lda} model.
```
Hvis du vil have applying i stedet for apply.

Reviewer 2: ```suggestion
Our work is similar to that of \citet{MetaLDA2017} since we work with meta information and are applying it to an \gls{lda} model.
```

> However, \citet{MetaLDA2017} learn a specific Dirichlet prior based on the topics based on the meta-information given in their dataset, which gives a specific topic distribution for each meta-information field and in turn affect the document topic distribution used with the \gls{lda}.

Reviewer: ```suggestion
However, \citet{MetaLDA2017} learn a specific Dirichlet prior based on the meta-information given in their dataset, which gives a specific topic distribution for each meta-information field and in turn affect the document-topic distribution used with the \gls{lda}.
```

> Instead of using the document topic distribution for drawing topic, we want to create a new meta-topic distribution which influences the drawn topics.

Reviewer: ```suggestion
Instead of using the document-topic distribution for drawing word topics, we want to create a new meta-topic distribution which influences the drawn topics.
```

An example is the author-topic model from \citet{author_topic_2012}, where they also create a distribution for each author.

> We would like to see what happens when you create a meta-topic distribution for any specific meta-information, like the author information, and see how it affects the resulting topics.

Reviewer: ```suggestion
We would like to see what happens when you create a meta-topic distribution for any specific meta-information, such as the author information, and see how it affects the resulting topics.
```

> Our work is similar to that of \citet{MetaLDA2017}, since we working with the meta information and combining it into an \gls{lda} model.
> However, \citet{MetaLDA2017} learn a specific Dirichlet prior over the topics based on the meta information given in their dataset, which gives a specific topic distribution for each meta information field and in turn affect the document topic distribution used with the \gls{lda}.
> Instead of using the document topic distribution for drawing topic, we want to create a new meta-topic distribution which influences the drawn topics.
> An example is the author-topic model from \citet{author_topic_2012}, where they also create a distribution for each author.
> We would like to see what happens when you create a meta-topic distribution for any specific meta information, like author, and see how it affect the resulting topics.
> The intuition behind this is to create new topic models, which describe the data in a different way using topics like meta data describe something about locations we can find location specific topics.

Author: ```suggestion
Our work is similar to that of \citet{MetaLDA2017}, since we working with the meta information and combining it into an \gls{lda} model.
However, \citet{MetaLDA2017} learn a specific Dirichlet prior based on the topics based on the meta-information given in their dataset, which gives a specific topic distribution for each meta-information field and in turn affect the document topic distribution used with the \gls{lda}.
Instead of using the document topic distribution for drawing topic, we want to create a new meta-topic distribution which influences the drawn topics.
An example is the author-topic model from \citet{author_topic_2012}, where they also create a distribution for each author.
We would like to see what happens when you create a meta-topic distribution for any specific meta-information, like the author information, and see how it affects the resulting topics.
The intuition behind this is to create new topic models, which describe the data in a different way using topics like metadata describe something about locations we can find location-specific topics.
```

> The intuition behind this is to create new topic models, which describe the data in a different way using topics like metadata describe something about locations we can find location-specific topics.

Reviewer: ```suggestion
The intuition behind this is to create new topic models, which describe the data in a different way using topics that are influenced by metadata.
For example, if the metadata describes something about geographic locations, location-specific topics will be generated.
```
Prøvede at ændre det lidt, da jeg ikke var sikker på hvad du prøvede at sige.
